metadata.rb

```diff
@@ -12,6 +12,8 @@
 depends          'chef-vault', '~> 1.3'
 depends          'ulimit', '~> 0.3'
 depends          'xml', '~> 1.2'
+# build-essential >=8.0.0 requires chef 12.5+
+depends          'build-essential', '< 8.0.0'
```

Comment:
chefspec was failing on ruby 2.1.6 build because of build-essential and reason why I didn't lock it in the Gemfile is mentioned in this [comment](https://github.com/cerner/cerner_splunk/pull/140#discussion_r101816360).

https://travis-ci.org/cerner/cerner_splunk/jobs/202454619